src/main.c

```diff
@@ -56,10 +56,18 @@ print_usage()
            "                (used as generation id for postgres)\n"
            "-f | -F : consumer / producer filename (only file)\n"
            "-l      : path to the log file\n"
+           "-V      : print version\n"
            "\n");
     exit(1);
 }
 
+void
+print_version()
+{
+    printf("schaufel version: %s\n\n", _SCHAUFEL_VERSION);
+    exit(1);
```

Comment:
Why not exit with zero? At this point we've successfully printed out the version.

Reply:
Yes, unfortunately `-h` is taken :)